server_src/services/database/server-settings.service.js

```diff
@@ -0,0 +1,311 @@
+const ServerSettingsDB = require("../../models/ServerSettings.js");
+
+const { isEmpty } = require("lodash");
+
+class ServerSettings {
+  currentServerSettings;
+
+  /**
+   * Returns the entire settings object from catch
+   * @returns {Object} Object from cache for all of OctoFarms settings
+   */
+  get entireSettingsObject() {
```

Comment:
Naming is weird to me. I'd name it `getAllSettingsGroups` (Groups, Sections, Children, whatever) to indicate that the settings are grouped and require traversing to get a specific setting